src/consts.py

```diff
@@ -0,0 +1,14 @@
+TITLE_MESSAGE = '\n\n--------WASHU TRANSFER COURSE FINDER--------'
+ZIP_MESSAGE = '\n\nTo find colleges offering courses near you, enter your zip code: '
+CONN_STRING = "host='ec2-18-222-149-129.us-east-2.compute.amazonaws.com' dbname='transfer_courses' user='guest' password='guest'"
+URL_STRING = "http://registrar.seas.wustl.edu/EVALS/evals.asp?school="
+NO_ZIP_MATCH_MESSAGE = "\nNo zip codes were found that match your input."
+TRY_AGAIN_MESSAGE = "\nWould you like to try again? Enter Y to try again or any other key to leave. "
+DEPT_CODE_MESSAGE = "\n\nEnter the WashU Department Code for the course you would like to receive"
+                                "credit for. \nOr, enter 'L' to see a list of deparments:  "
+COURSE_NUMBER_MESSAGE = "\n\nEnter the WashU Course Number for the course you would like to"
+                                    "\nreceive credit for: "
+NO_COURSES_MESSAGE = "\nNo courses were found that match your input."
+PICK_SCHOOL_MESSAGE = "\nWhat school would you like to see courses for? Enter the number or enter Q to exit: "
+SRC = './src/'
+MAIN = "__main__"
```

Comment:
I think this should be reordered so that connection/url messages are together, directions are together, and errors are together.

Reply:
good idea for reodering so it stays organized!

Reply:
plus, lets label our sections with comments